_plugins/author.rb

```diff
@@ -3,6 +3,7 @@ class AuthorTag < Liquid::Tag
     def initialize(tag_name, author, tokens)
       super
       @author = author.strip
+      @baseurl = Jekyll.sites[0].config['baseurl']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.